This is not a proper fix, but a workaround that is needed on HEAD.  (Have
to check glibc-2_5-branch.)  Reported on
<http://savannah.gnu.org/bugs/?18218>.

---
 resolv/res_send.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/resolv/res_send.c
+++ b/resolv/res_send.c
@@ -102,6 +102,8 @@
 #define MAXPACKET       65536
 #endif
 
+#undef MSG_NOSIGNAL
+#define MSG_NOSIGNAL	0
 
 /* From ev_streams.c.  */
 
